anagram.cpp

```diff
@@ -1,13 +1,44 @@
 #include "anagram.h"
+#include <string>
+#include <vector>
+#include <bits/stdc++.h>
+#include <algorithm>
+
+using namespace std;
 
 bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2) {
-    //Fill the correct implementation here
+    
+    string word1Copy = word1;
+    string word2Copy = word2;
+    
+    word1Copy.erase(remove(word1Copy.begin(), word1Copy.end(), ' '), word1Copy.end());
```

Comment:
Whenever space has to be removed, it is common to remove punctuations and other whitespace characters. This may be needed to satisfy some of the other examples as cited above.

Reply:
Yes, @sudeeprp. I didn't consider that case. Now I am removing all special characters.